lib/elixir/lib/file.ex

```diff
@@ -461,7 +461,7 @@ defmodule File do
   error will be returned.
   """
   @spec cp(Path.t, Path.t, (Path.t, Path.t -> boolean)) :: :ok | {:error, posix}
-  def cp(source, destination, callback \\ fn(_, _) -> true end) do
+  def cp(source, destination, callback \\ &path_differs?/2) do
```

Comment:
Given that returning true would create an empty file, which is a bug, i
really don't think we want to allow the user to have this option. :)